mkdocs.yml

```
@@ -850,7 +852,7 @@ plugins:
       xdr/features/investigate/dork_language.md: xdr/features/investigate/events_query_language.md
       operation_center/data_collection/ingestion_methods/sekoiaio.md: integration/categories/endpoint/sekoiaio.md
 - redoc
-- intakes_by_uuid
+# - intakes_by_uuid
```

This plugin must be preserved
```suggestion
 - intakes_by_uuid
```

---

outch nice catch thanks !